Tweaks emergency shuttle control console

Kasuobes opened the pull request:
The control console will no longer accept ID authorizations during crew transfers, functionality remains the same for evacuations.

The reason for this change is that there are no circumstances where overriding the autopilot during a crew transfer isn't breaking RP. If people launch the shuttle early, they get bwoinked; if they hold the shuttle, they get bwoinked. In both cases overriding the autopilot during a crew transfer is unrealistic and pointless, might as well stop the practice entirely.

Kasuobes committed:
Tweaks emergency shuttle control console

The control console will no longer accept ID authorizations during crew transfers, functionality remains the same for evacuations.

The reason for this change is that there are no circumstances where overriding the autopilot during a crew transfer isn't breaking RP. If people launch the shuttle early, they get bwoinked; if they hold the shuttle, they get bwoinked. In both cases overriding the autopilot during a crew transfer is unrealistic and pointless, might as well stop the practice entirely.

Raptor1628 commented:
Can admins still override it? One big reason for allowing overrides, even on transfers, is if the shuttle gets held up by depowered airlocks or something of that nature.

atlantiscze commented:
Not a fan of this. There are situations where override may be in place. Such as, when an imminent emergency occurs while the shuttle is already enroute (relatively rare thing) or, as mentioned, when the shuttle's automatic sequence gets stuck.

Kasuobes commented:
The force launch button in fact works without requiring any authorization (anyone can press the button and force launch), it becomes available when the shuttle begins the undocking sequence, so if airlocks become stuck or unpowered the crew can still launch the shuttle manually, but only after the countdown timer has ended and the shuttle is undocking.

Admins can still adjust the shuttle timer in the check antag panel to make the shuttle launch sooner or later, for example setting ETL to 0 seconds to immediately begin undocking.

Alternatively I can set it so the override become available during red alert or higher, since two head IDs are required to set red alert and override the shuttle anyways. This is a very rare case where red alert is called during a crew transfer so I don't see too much of an abuse potential here.

Kasuobes committed:
Allows override during red alert or higher

In rare cases where a red alert is declared during a crew transfer, the shuttle controls can be overridden.

Techhead0 commented on code/modules/shuttles/shuttle_emergency.dm:
```diff
@@ -126,7 +126,7 @@
 		return 0	//don't need any more
 
-	if (!emergency_shuttle.evac)
+	if (!emergency_shuttle.evac || security_level < SEC_LEVEL_RED)
```
Typo.
This should either be:
`if (!emergency_shuttle.evac && security_level < SEC_LEVEL_RED)`
or
`if (!(emergency_shuttle.evac || security_level >= SEC_LEVEL_RED))`.

Kasuobes replied:
Thanks, fixed.

Kasuobes commented:
Bump for review.
